yowsup/layers/protocol_profiles/protocolentities/iq_privacy_get.py

```diff
@@ -0,0 +1,27 @@
+from yowsup.layers.protocol_iq.protocolentities import IqProtocolEntity
+from yowsup.structs import ProtocolTreeNode
+
+'''
+<iq xmlns="privacy" type="get" id="{{IQ_ID}}">
+<privacy>
+</privacy>
+</iq>
+'''
+
+class GetPrivacyIqProtocolEntity(IqProtocolEntity):
+    XMLNS = "privacy"
```

Comment:
This confused me a bit as it's not being used as namespace, so I changed variable name to NODE_PRIVACY